config.jsonc

```diff
@@ -60,7 +60,7 @@
         },
         {
             "type": "custom",
-            "format": "|──────────────\u001b[1mUptime / Age\u001b[0m──────────────────|"
+            "format": "├─────────────── \u001b[1mUptime / Age\u001b[0m ───────────────┤"
```

Comment:
This left aligns like to the above 2 categories, and make sage and uptime have the same order as the listing itself
"format": "├─────────── \u001b[1mAge & Uptime\u001b[0m ───────────────────┤"